README.md

```diff
@@ -1,11 +1,12 @@
 Formatted Text
 =============
+An imperative approach to multi-line formatted text; its 
+[data model](explainer-datamodel.md), [rendering](explainer-rendering.md), and
+[metrics](explainer-metrics.md).
```

Comment:
use text-metrics rather than metrics, so that people dont confuse it with some API to get perf metrics.

Reply:
Good catch. Done.

Reply:
See: https://github.com/WICG/canvas-formatted-text/commit/ee6113391b5048638e5c58d4d9da60f022481387